addons/hr_timesheet/views/project_portal_templates.xml

```diff
@@ -45,8 +45,8 @@
 
     <template id="portal_tasks_list_inherit" inherit_id="project.portal_tasks_list" name="Portal: My Tasks with Timesheets">
         <xpath expr="//t[@t-call='portal.portal_table']" position="inside">
-            <t t-set="timesheet_ids" t-value="task.sudo().timesheet_ids"/>
-            <t t-set="is_uom_day" t-value="timesheet_ids._is_timesheet_encode_uom_day()"/>
+                <t t-set="timesheet_ids" t-value="task.sudo().timesheet_ids"/>
+                <t t-set="is_uom_day" t-value="timesheet_ids._is_timesheet_encode_uom_day()"/>
```

Comment:
This diff should be in the second commit instead of the first one. 🙂

Reply:
not done...

Reply:
Done changes :+1: